sliceguard/modeling.py

```diff
@@ -107,3 +109,96 @@ def fit_classification_regression_model(
         raise RuntimeError("Could not run inference. Not valid task given.")
 
     return y_preds, y_probs, classes
+
+
+def _check_training_imports():
```

Comment:
Use a centralized check, probably move import checks to own module. Do not duplicate this from embeddings.